src/dune_rules/coq_rules.ml

```diff
@@ -394,6 +396,20 @@ let coq_modules_of_theory ~sctx lib =
   let coq_sources = Dir_contents.coq dir_contents in
   Coq_sources.library coq_sources ~name
 
+let ffi_rule ~dir ~coqffi ~expander (s : string) : Action.t Build.With_targets.t =
+  let ml_lib_name, coq_module_name = Coq_stanza.Theory.ffi_parse_name s in
+  let coq_v_name = Path.Build.relative dir (coq_module_name ^ ".v") in
+
+  (* Get the path to the cmi file *)
+  (* We need help here! The below fails with wrong dune *)
+  let expand_expr = String_with_vars.make_var Loc.none ~payload:ml_lib_name "cmi" in
+  let cmi_path = Expander.expand_path expander expand_expr in
```

Comment:
This doesn't work, we also tried:
```ocaml
  let cmi_path =
    let open Result.O in
    let* ml_lib = Lib.DB.resolve lib_db (Loc.none, Lib_name.of_string ml_lib_name) in
    let info = Lib.info ml_lib in
    let obj_dir = Obj_dir.public_cmi_dir (Lib_info.obj_dir info) in
    Result.Ok (Path.relative obj_dir (coq_module_name ^ ".cmi"))
```
which works but it doesn't compute the right cmi path for wrapped libs.